src/JsonSchema/SchemaFactory.php

```diff
@@ -69,7 +69,7 @@ public function addDistinctFormat(string $format): void
     public function buildSchema(string $className, string $format = 'json', string $type = Schema::TYPE_OUTPUT, ?string $operationType = null, ?string $operationName = null, ?Schema $schema = null, ?array $serializerContext = null, bool $forceCollection = false): Schema
     {
         $schema = $schema ?? new Schema();
-        if (null === $metadata = $this->getMetadata($className, $type, $operationType, $operationName, $serializerContext)) {
+        if (!class_exists($className) || null === $metadata = $this->getMetadata($className, $type, $operationType, $operationName, $serializerContext)) {
```

Comment:
I don't think that it's a good idea to hide the fact that a non-existing class went through here.

Reply:
How do we end up with a bad class? Isn't this because of corruption in `phpdocumentor/reflection-docblock` and `symfony/property-info`?

Reply:
Yes but I first tried to fix it in Api Platform before I could trace it back to the problem in phpdocumentor used by property-info...